src/main/scala/elo/LeaderboardService.scala

```diff
@@ -1,9 +1,28 @@
 package elo
+import scala.util.Try
 
 class LeaderboardService(playerService: PlayerService) {
 
-  def getLeaderboard(numTopPlayers: Int = 3): Seq[Player] = {
-    playerService.findAllPlayers.sortBy(-_.score).take(numTopPlayers)
+  val defaultLeaderboardSize = 3
```

Comment:
Naming convention for constant variables is upper camel case, so this would be DefaultLeaderboardSize.